test/verify/test_convolution_backwards_3d.cpp

```diff
@@ -39,7 +39,7 @@ struct test_deconv_3d : verify_program<test_deconv_3d>
             mm->add_parameter("w", migraphx::shape{migraphx::shape::float_type, {1, 1, 3, 3, 3}});
         mm->add_instruction(
             migraphx::make_op(
-                "deconvolution",
+                "convolution_backwards",
```

Comment:
Dilation 1 is default. We should probably modify or add additional test with non-default value of all three attributes strides, dilation, padding together.